### DISCUSSION TITLE
How do I install it once I download it (Can't find the installer file within the folder)

### DISCUSSION THREAD

**Discussion by samxyx** [AntiMicroX/antimicrox#768] [Q&A] [answered]
Trying to install it on Windows 11. I downloaded it and extracted the files form the zip folder, but I can't seem to find the installer. The "ReadMe" on the help page just said to "install it". Sorry if this is a dumb question, but I can't figure it out.

**Comment by pktiuk** [comment]
Everything is on main page of project 
 https://github.com/AntiMicroX/antimicrox#windows

> Just download antimicrox-X.X.X-AMD64.exe from Release site and install it.

**Reply by samxyx** [reply, marked as answer]
Thanks. If anyone else is having the same problem I was, make sure you are looking at the release page (https://github.com/AntiMicroX/antimicrox/releases/tag/3.3.3) and not the description page (https://github.com/AntiMicroX/antimicrox#windows). Under the release page scroll down till you see assets and download the file that correlates to your system.

I thought the description page was the release page and was mistaking the source code files for the actual installer.
